### DISCUSSION TITLE
Will the TAPO RV30 Plus be added ?

### DISCUSSION THREAD

**Discussion by bierlaagh** [petretiandrea/home-assistant-tapo-p100#612] [Ideas]
I just received a TAPO RV30 Plus vacuum and Mop robot.
is there a change or opportunity to add it?

**Comment by GSzabados** [comment]
FYI - https://community.home-assistant.io/t/tp-link-tapo-rv10-rv30/581187/12

**Comment by jan-tdy** [comment]
Probably not. 
Mainly because other integrations(like mine https://github.com/jan-tdy/TapoVac-ADV) already have it
